boards/arm/stm32l4r9i_disco/stm32l4r9i_disco_defconfig

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32L4X=y
+CONFIG_SOC_STM32L4R9XX=y
+
+# Enable UART
+CONFIG_SERIAL=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable GPIO
+CONFIG_GPIO=y
+
+# Enable clocks
+CONFIG_CLOCK_CONTROL=y
+
+# Enable pin controller
+CONFIG_PINCTRL=y
```

Comment:
Please add :
```
# Enable MPU
CONFIG_ARM_MPU=y
# Enable HW stack protection
CONFIG_HW_STACK_PROTECTION=y
```
Should also be added on the board you used as example if not already set (make it a dedicated commit)

Reply:
Example boards have this config, I just didn't copy it as I didn't check if MCU had a MPU.
Done.